vhost-user-backend/src/bitmap.rs

```diff
@@ -0,0 +1,602 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Other files in `vhost-user-backend/src` have a Copyright with the rights holder, should we add one here too? (above the license I mean)

Reply:
Sorry, I meant if you want to put your company copyright here.

In https://www.apache.org/licenses/LICENSE-2.0#apply there is the `name of copyright owner`, I don't know about "All rights reserved":
```
Copyright [yyyy] [name of copyright owner]

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
```